setup.py

```diff
@@ -2,18 +2,18 @@
 
 
 setuptools.setup(
-    name="executor-image-normalizer",
+    name="executors-evaluator-ranking",
     version="2.0",
```

Comment:
This is not JIna version. Instead, it is just the version of the executor.
```suggestion
    version="1",
```